timelock/program/src/processor/process_create_empty_timelock_config.rs

```diff
@@ -0,0 +1,61 @@
+//! Program state processor
+
+use crate::{
+    state::timelock_config::TimelockConfig,
+    state::timelock_program::TimelockProgram,
+    utils::{assert_initialized, assert_token_program_is_correct, create_account_raw},
+};
+use solana_program::{
+    account_info::{next_account_info, AccountInfo},
+    entrypoint::ProgramResult,
+    pubkey::Pubkey,
+};
+
+/// Create empty timelock config
+pub fn process_create_empty_timelock_config(
+    program_id: &Pubkey,
+    accounts: &[AccountInfo],
+) -> ProgramResult {
+    let account_info_iter = &mut accounts.iter();
+    let timelock_config_account_info = next_account_info(account_info_iter)?;
+    let program_to_tie_account_info = next_account_info(account_info_iter)?;
+    let governance_mint_account_info = next_account_info(account_info_iter)?;
+    let council_mint_account_info = next_account_info(account_info_iter)?;
+    let payer_account_info = next_account_info(account_info_iter)?;
+    let timelock_program_account_info = next_account_info(account_info_iter)?;
+    let timelock_program_info = next_account_info(account_info_iter)?;
```

Comment:
This was a bit confusing -- you don't need to pass this in thankfully, and you can just the `program_id` from the processor function, and then not pass it in to `create_account_raw`

Reply:
I actually am going to drop the timelock "account" entirely. I realized that the creation of this stemmed from me sort of following the token-lending pattern a bit too closely. I don't think I really need it if I just store the token_program on each set. This confusion is really an artifact of this being my first smart contract and learning. One of the refactorings Seb is doing in these next couple weeks is going to be to rip out timelock_program_account from this system entirely, it's like an appendix!

Reply:
You can definitely drop the `timelock_program_account` -- this comment is about the `timelock_program_info` being passed in

Reply:
Done